src/lib/pipeline.ml

```diff
@@ -392,6 +402,8 @@ let rec to_file_prefix:
       sprintf "%s-%s-%s" prev
         vc.Variant_caller.name
         vc.Variant_caller.configuration_name
+    | Rna_hla_typer s ->
+      sprintf "RNA_HLA_typer-%s" (to_file_prefix ?read s)
```

Comment:
To the extend that something needs to be "future-proof", there will exist a commit that will be aligned with the code that it is generating this JSON. So a "deserialization", to the extend that you may want one (even though, AFAIK, one currently doesn't exist), can be constructed. But since all you want is just more verbose and precise names and json, sure.

Addressed.